apis/notificationserviceapi/notificationserviceapi.yaml

```diff
@@ -0,0 +1,788 @@
+                  ts: '2021-08-11 21:20:28:919+0530'
+                  params:
+                    resmsgid: null
+                    msgid: "ed1c0b7c-8f10-4a35-9f42-93966d8d97f6"
+                    err: null
+                    status: null
+                    errmsg: null
+                  responseCode: OK
+                  result:
+                      response: SUCCESS
+        '400':
+            description: BAD REQUEST. The "Send Notifications" operation failed ! The possible
+              reason for failure is that you may have missed providing input for a
+              mandatory parameter.
+            content:
+              application/json:
+                schema:
+                 $ref: ""
+                example:
+                  id: v2.notification.send
+                  ver: null
+                  ts: 2021-08-11 21:20:28:919+0530
+                  params:
+                    resmsgid: null
+                    msgid: "ed1c0b7c-8f10-4a35-9f42-93966d8d97f6"
+                    err: INVALID_REQUESTED_DATA
+                    status: null
+                    errmsg: '#/notifications/0: required key [type] not found'
+                    responseCode: CLIENT_ERROR
+                    result: {}
+        '401':
+            description: Unauthorized User
+            content:
+              application/json:
+                schema:
+                 $ref: ""
+                example:
+                  id: v2.notification.send
+                  ver: null,
+                  ts: 2021-08-11 21:30:57:395+0530
+                  params:
+                    resmsgid: null,
+                    msgid: "ed1c0b7c-8f10-4a35-9f42-93966d8d97f6"
+                    err: unauthorized
+                    status: null,
+                    errmsg: you are an unauthorized user
+                    responseCode: UNAUTHORIZED
+                    result: {}
+         
+        '500':
+           description: "'INTERNAL SERVER ERROR. We track these errors automatically and
+             try to set it right at the earliest. Try refreshing the page. If the
+             problem persists contact us at info@sunbird.org.'"
+           content:
+             application/json:
+               schema:
+                 $ref: ""
+               example:
+                 type: object
+                 id: v2.notification.send
+                 ver: v1
+                 ts: 2020-12-01 10:49:01:948+0000
+                 params:
+                  resmsgid: 
+                  msgid: "ed1c0b7c-8f10-4a35-9f42-93966d8d97f6"
+                  err: null
+                  status: null
+                  errmsg: "SERVER_ERROR"
+                 responseCode: SERVER_ERROR
+                 result: {}           
+  "/notification/v1/feed/read/{userId}":
+     get:
+      summary: Read User Notification Feed
+      description: >-
+        This API is used for getting in add notification of the user, it is based on the unique
```

Comment:
change to: This API is used to get In-App notification for a userId, user Id uniquely identifies a user.